pkg/controller/datavolume-controller.go

```diff
@@ -2836,6 +2836,7 @@ func (r *DatavolumeReconciler) validateCloneAndSourcePVC(datavolume *cdiv1.DataV
 	if err != nil {
 		// Clone without source
 		if k8serrors.IsNotFound(err) {
+			r.recorder.Eventf(datavolume, corev1.EventTypeWarning, ErrUnableToClone, "Source pvc %s not found", datavolume.Spec.Source.PVC.Name)
```

Comment:
Overall looks very good, but wouldn't this event be a little bit redundant? We are just recording a very similar event in the line below

Reply:
I just moved it from the get clone strategy to here since it didnt seem related there, you added the event so you proabably know best if its redundant, if it is Ill delete :)

Reply:
The first event says `"Source pvc %s not found"` while the second one is `"The source PVC doesn't exist"`, so I think they are too similar to record them at the same time. Maybe it's a good idea to merge the two so the second event also displays the PVC name, but I'd just use one.

Reply:
done